docs/AboutUs.md

```diff
@@ -17,7 +17,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](team/johndoe.md)]
 
 * Role: Team Lead
-* Responsibilities: Overall project coordination.
+* Responsibilities: Responsible for overall project coordination.
```

Comment:
Would having both the words "Responsibilities" and "Responsible" be repetitive?
Same as line 62
```suggestion
* Responsible for overall project coordination.
```

Reply:
Yes, that's definitely clearer. Fixed!